src/bun.js/test/jest.zig

```diff
@@ -1047,6 +1048,26 @@ pub const DescribeScope = struct {
         return null;
     }
 
+    pub fn runBeforeAll(
+        this: *DescribeScope,
+        globalObject: *JSGlobalObject,
+    ) ?JSValue {
+        if (this.parent) |p| {
+            if (p.runBeforeAll(globalObject)) |err| {
+                return err;
+            }
+        }
+
+        if (this.needs_before_all) {
```

Comment:
I believe the `needs_before_all` check is redundant, but was mimicking the `needs_before_each` to ensures we dont accidentally run a `beforeAll` twice